core/src/main/java/io/onetable/avro/AvroSchemaConverter.java

```diff
@@ -414,6 +414,7 @@ private Schema fromOneSchema(OneSchema oneSchema, String currentPath) {
                       Schema.createFixed(oneSchema.getName(), oneSchema.getComment(), null, size)),
               oneSchema);
         }
+      case UUID:
```

Comment:
I don't know how to handle toOneSchema here, for Avro doesn't have a UUID type, the same as DeltaLake.

Reply:
Fixed size byte array makes the most sense in my opinion